nf_core/pipeline-template/subworkflows/local/utils_nfcore_pipeline_completion/meta.yml

```
@@ -0,0 +1,39 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/nf-core/modules/master/subworkflows/yaml-schema.json
+name: "pipeline_completion"
+description: Notify user of pipeline completion
+keywords:
+  - notification
+  - email
+  - completion
+components: []
+input:
+  - email:
+      type: string
+      description: Email address
+  - email_on_fail:
+      type: string
+      description: Email address sent on pipeline failure
+  - plaintext_email:
+      type: boolean
+      description: Send plain-text email instead of HTML
+  - outdir:
+      type: string
+      description: Path to output directory where results will be published
+  - monochrome_logs:
+      type: boolean
+      description: Disable ANSI colour codes in log output
+  - hook_url:
+      type: string
+      description: Hook URL for notifications
+  - multiqc_report:
+      type: string
+      description: Path to MultiQC report
+output:
+  - dummy_emit:
+      type: boolean
+      description: |
+        Dummy emit to make nf-core subworkflows lint happy
+authors:
+  - "@mirpedrol"
```

let's fake the user and use the nf-core-bot for this, I didn't create this subworkflows 😄 
```suggestion
  - "@nf-core-bot"
```
And the same for the others